test_runner/regress/test_timeline_archive.py

```diff
@@ -32,11 +30,15 @@ def test_timeline_archive(neon_simple_env: NeonEnv):
 
     assert exc.value.status_code == 404
 
-    # for non existing tenant:
+    # for a non existing tenant:
     invalid_tenant_id = TenantId.generate()
+    if unsharded:
+        not_found_pattern = f"NotFound: tenant {invalid_tenant_id}"
+    else:
+        not_found_pattern = "NotFound: Tenant not found"
```

Comment:
same here, why can't we unify the two?

Reply:
If you search for that error message you see it's the same over all http handlers in the storage controller. I think it's fine to not unify.

Reply:
hmm yeah sounds like something we should align, but for this I think we could make due with plain `NotFound: [tT]enant.*`.

Reply:
addressed.